ocp_cleanup.sh

```diff
@@ -10,3 +10,4 @@ VOL_POOL=$(sudo virsh vol-pool "/var/lib/libvirt/images/${CLUSTER_NAME}-bootstra
 sudo virsh vol-delete "${CLUSTER_NAME}-bootstrap.ign" --pool "${VOL_POOL}"
 rm -rf ocp
 sudo rm -rf /etc/NetworkManager/dnsmasq.d/openshift.conf
+rm -f ocp/install-config.yaml
```

Comment:
but we already remove the ocp directory 2 lines up, is that not working for some reason?

Reply:
You're right, I don't know why I didn't pay closer attention to the line you mentioned.  I was clearly looking at one file and fixing another.  Thanks for the catch.

Reply:
Ok so I think this PR can be closed, right?